cse/src/main/java/com/adyen/checkout/cse/CardEncryptor.kt

```diff
@@ -8,19 +8,15 @@
 
 package com.adyen.checkout.cse
 
-import com.adyen.checkout.cse.internal.BaseCardEncrypter
-import com.adyen.checkout.cse.internal.ClientSideEncrypter
-import com.adyen.checkout.cse.internal.DateGenerator
-import com.adyen.checkout.cse.internal.DefaultCardEncrypter
-import com.adyen.checkout.cse.internal.DefaultGenericEncrypter
+import com.adyen.checkout.cse.internal.CardEncryptorFactory
 
 /**
  * Allows the encryption of card data to be sent to Adyen's APIs.
  * Use this class with the custom card integration.
  */
-object CardEncrypter {
+object CardEncryptor {
```

Comment:
Is this a breaking change?

Reply:
Yes, back then I hoped to merge it before v4. Let's revert it